pom.xml

```diff
@@ -1,8 +1,8 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>zyenyo.Zyenyo</groupId>
+  <groupId>com.seizoxu.zyenyo</groupId>
```

Comment:
The original was a placeholder, but so is this new one, because I can't decide on what I should change it to. In the meanwhile, some standardisation made sense.